src/gui/graphdesigner.cpp

```diff
@@ -42,7 +42,7 @@
 
 namespace evoplex {
 
-GraphDesigner::GraphDesigner(MainGUI* mainGUI, QWidget* parent)
+GraphDesigner::GraphDesigner(MainGUI* mainGUI, GraphDesignerPage* parent)
```

Comment:
We should not make this class strongly tied to the `GraphDesignerPage`. Please, revert this line to take a `QWidget` instead of the `GraphDesignerPage`